packages/oui-action-menu/_mixins.less

```diff
@@ -24,11 +24,13 @@
       color: @oui-action-menu-text-color;
 
       &:hover, &:focus {
-        background-color: @oui-action-menu-item-active-background-color;
-        color: @oui-action-menu-item-active-text-color;
+        &:not(:disabled) {
```

Comment:
First, about it, but would ```&:not(disabled):hover, &:not(disabled):focus``` also work.

And for my personal knowledge, is there a difference between ```:not(:disabled)``` and ```:enabled```?

Reply:
Yes `&:not(:disabled):hover, &:not(:disabled):focus` also works but nested version is shorter.
`:enabled` and `:not(:disabled)` are the same ;)